core/amber/src/main/scala/edu/uci/ics/texera/workflow/operators/source/scan/csv/CSVScanSourceOpExec.scala

```diff
@@ -17,11 +19,13 @@ class CSVScanSourceOpExec private[csv] (
     customDelimiter: Option[String],
     hasHeader: Boolean,
     schemaFunc: () => Schema
-) extends SourceOperatorExecutor {
+) extends SourceOperatorExecutor
+    with CheckpointSupport {
```

Comment:
please mention this change in PR description.

Reply:
done